rest/taskana-rest-spring/src/test/resources/application.properties

```diff
@@ -25,13 +25,13 @@ taskana.ldap.userSearchFilterValue=person
 taskana.ldap.userFirstnameAttribute=givenName
 taskana.ldap.userLastnameAttribute=sn
 taskana.ldap.userIdAttribute=uid
-taskana.ldap.groupSearchBase=cn=groups
+taskana.ldap.groupSearchBase=
```

Comment:
What's with the user search base?
And what about our other X configs:
* rest/taskana-rest-spring-example-boot/src/main/resources/application-postgres.properties
* rest/taskana-rest-spring-example-boot/src/main/resources/application.properties
* rest/taskana-rest-spring-example-common/src/test/resources/application.properties
* rest/taskana-rest-spring-example-wildfly/src/main/resources/application-postgres.properties
* rest/taskana-rest-spring-example-wildfly/src/main/resources/application.properties
* rest/taskana-rest-spring-example-wildfly/src/test/resources/application.properties
* rest/taskana-rest-spring/src/test/resources/application.properties
* ADAPTER: taskana-adapter-camunda-wildfly-example/src/main/resources/application.properties

Reply:
We can decide about the other properties. They don't need to be exactly the same. Do you prefer to have the same settingseverywhere?

Reply:
Again, let's talk about this :)

Reply:
Since the properties are different already, I will not touch this point in this ticket.